### DISCUSSION TITLE
Is the new TeX math only available for markdown in README.md?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23439] [Other Feature Feedback, Questions, & Ideas]
<p>I’m pretty sure I’m missing something, but I can’t see what exactly. It was <a href="https://github.blog/2022-05-19-math-support-in-markdown/" rel="noopener nofollow ugc">announced</a>, in GitHub’s own words, that “you can use the $ and $$ delimiters natively in Markdown on GitHub to insert math expressions in TeX and LaTeX style syntax”, pretty much full stop. And yet I only seem to get it to work for <a href="https://github.com/zagstex/zagstex.github.io#readme" rel="noopener nofollow ugc">README.md</a> or when I preview something <a href="https://github.com/zagstex/zagstex.github.io/blob/adbe4e8941c63cc4c35573d86d73e9209733c25b/index.md" rel="noopener nofollow ugc">on the repository</a>, but not on <a href="https://zagstex.github.io" rel="noopener nofollow ugc">my github.io page</a> (consider the first sentence).</p>
<p>Any ideas? Thanks in advance, cheers!</p>

**Comment by yoannchaudet** [comment]
<p>Hello <a class="mention" href="/zagstex">@zagstex</a> and welcome here,</p>
<p>GitHub Pages was unfortunately not part of this announcement. Pages sites are built using <a href="https://jekyllrb.com/">Jekyll</a> by default. It is a static site generator that is independent (and open source).</p>
<p>We have an incoming Pages feature (~ end of the month) that will allow for better support of various static site generators, including full control over Jekyll plugins one decide to build a site with (today, for security reasons related to our build infrastructure, <a href="https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll#plugins">we limit the plugins we allow</a>).</p>
<p>To answer your question: no we don’t support the same syntax in Pages that <a href="http://GitHub.com">GitHub.com</a> is supporting. We will very soon open up Pages further plugins such as <a href="https://github.com/jeffreytse/jekyll-spaceship#2-mathjax-usage" class="inline-onebox">GitHub - jeffreytse/jekyll-spaceship: 🚀 A Jekyll plugin to provide powerful supports for table, mathjax, plantuml, mermaid, emoji, video, audio, youtube, vimeo, dailymotion, soundcloud, spotify, etc.</a> can be used more easily.</p>
